python_modules/dagster/dagster/_api/snapshot_partition.py

```diff
@@ -36,9 +34,7 @@ def sync_get_external_partition_names_grpc(
         api_client.external_partition_names(
             partition_names_args=PartitionNamesArgs(
                 repository_origin=repository_origin,
-                job_name=job_name,
```

Comment:
we need to still set this right? since there is a live agent version (1.8.2) that has it as required?

Reply:
ah did not realize. added this back in